index.bs

```diff
@@ -11346,6 +11354,195 @@ The [=remote end steps=] given |session| and |command parameters| are:
+
+1. Let |type| be |extension data spec|["<code>type</code>"].
+
+1. If installing a web extension using |type| isn't supported return [=error=] with [=error code=] [=unsupported operation=].
+
+1. In the following list of conditions and associated steps, run the first set of steps for which the associated condition is true:
+
+   <dl>
+    <dt>|type| is the string "<code>path</code>"
+    <dd>
+      1. Let |path| be |extension data spec|["<code>path</code>"].
+      1. Let |locator| be a [=directory locator=] with [=file system locator/path=] |path| and [=file system locator/root=] corresponding to the root of the file system.
+      1. Let |entry| be [=locate an entry=] given |locator|.
+
+    <dt>|type| is the string "<code>archivePath</code>"
+    <dd>
+      1. Let |archive path| be |extension data spec|["<code>path</code>"].
+      1. Let |locator| be a [=file locator=] with [=file system locator/path=] |archive path| and [=file system locator/root=] corresponding to the root of the file system.
+      1. Let |archive entry| be [=locate an entry=] given |locator|.
+      1. If |archive entry| is null, return null.
+      1. Let |bytes| be |archive entry|'s [=file entry/binary data=].
+      1. Let |entry| be the result of [=trying=] to [=extract a zip archive=] given |bytes|.
+
+    <dt>|type| is the string "<code>base64</code>"
+    <dd>
+      1. Let |bytes| be [=forgiving-base64 decode=] |extension data spec|["<code>path</code>"].
+      1. If |bytes| is failure, return null.
+      1. Let |entry| be the result of [=trying=] to [=extract a zip archive=] given |bytes|.
+      1. If |entry| is failure, return null.
+
+   </dl>
+
+1. Return |entry|.
+
+</div>
+
+<div algorithm="remote end steps for webExtension.install">
+The [=remote end steps=] with |command parameters| are:
+
+1. If installing web extensions isn't supported return [=error=] with error code [=unsupported operation=].
+
+1. Let |extension data spec| be |command parameters|["<code>extensionData</code>"].
+
+1. Let |extension directory entry| be the result of [=trying=] to [=expand a web extension data spec=] with |extension data spec|.
+
+1. If extension directory entry| is null, return [=error=] with [=error code=] [=invalid web extension=].
+
+1. Perform implementation defined steps to install a web extension from |extension directory entry|. If this fails, return [=error=] with [=error code=] [=invalid web extension=]. Otherwise let |extension id| be the unique identifier of the newly installed web extension.
+
+1. Let |result| be a [=/map=] matching the
+   <code>webExtension.InstallResult</code> production with the
+   <code>extension</code> field set to |extension id|.
+
+1. Return [=success=] with data |result|.
+
+</div>
+
+
+#### The webExtension.uninstall Command #### {#command-webExtension-uninstall}
+
+The <dfn export for=commands>webExtension.uninstall</dfn> command uninstalls a web extension for the browser.
```

Comment:
@jgraham  Should this reference remote end instead of the browser?
```suggestion
The <dfn export for=commands>webExtension.uninstall</dfn> command uninstalls a web extension for the [=remote end=].
```

Reply:
Yes, it should be the same as for the other command.